app/common/common-module.js

```diff
@@ -41,6 +41,7 @@ angular.module('ushahidi.common', [
 .controller('navigation', require('./controllers/navigation.js'))
 .controller('PageMetadata', require('./controllers/page-metadata.js'))
 .controller('notifier', require('./controllers/notifier.js'))
+.controller('intercom', require('./controllers/intercom.js'))
```

Comment:
Can we just load intercom outside angular (in a global) but then have angular send the user details on login/logout?